src/broker/cas.c

```diff
@@ -124,6 +124,10 @@ static void set_db_connection_info (void);
 static void clear_db_connection_info (void);
 static bool need_database_reconnect (void);
 
+extern int cas_init_ssl (int);
```

Comment:
how about change order of definition ?
extern bool ssl_client;
extern int cas_init_ssl (int);
extern void cas_ssl_close (int client_sock_fd);

Reply:
ok! I will accept it.